panda/src/putil/bamEnums.h

```diff
@@ -69,6 +69,12 @@ class EXPCL_PANDA_PUTIL BamEnums {
     BTM_basename,
     BTM_rawdata
   };
+
+  // Level of compression for textures in the .bam
+  enum BamTexCompressionLv{
+    BTC_off,
+    BTC_on,
```

Comment:
I would prefer BTC_zlib over BTC_on to make it easier to add other formats in the future (if desired).